kong/api/crud_helpers.lua

```diff
@@ -5,16 +5,43 @@ local app_helpers = require "lapis.application"
 
 local _M = {}
 
+--- Will look up a value in the dao.
+-- Either by `id` field or by the field named by 'alternate_field'. If the value is NOT
+-- a uuid, then by the 'alternate_field'. If it is a uuid then it will first try
```

Comment:
pedantic note: "a uuid" to me implies that the string we found is a UUID we generated. this could be clarified by noting the string "looks like a uuid" as opposed to is "_a_ uuid"